### PR TITLE
[Merged by Bors] - chore(Analysis): fix mathlib3 names; automated fixes

### PR DESCRIPTION
**Pull request by grunweg**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by grunweg** [comment]
Looking over the diff:
- I understand where the exp -> NormedSpace.exp change comes from; I'm not sure if it's actually better. Happy to revert those manually.
- False negative (my script so far missed one): `continuous_linear_map.adjoint` should also be pointed out

**Comment by kmill** [comment]
Thanks! I think `NormedSpace.exp` is good since it creates a link in the documentation.

bors r+

**Comment by grunweg** [comment]
Thank you for the fast review!

File: Mathlib/Analysis/BoxIntegral/Partition/Filter.lean [hunk 1]
```diff
@@ -34,15 +34,15 @@ the corresponding integral, or in the proofs of its properties. We equip
 
 The structure `BoxIntegral.IntegrationParams` has 3 boolean fields with the following meaning:
 
-* `bRiemann`: the value `true` means that the filter corresponds to a Riemann-style integral, i.e.
+* `bRiemann`: the value `True` means that the filter corresponds to a Riemann-style integral, i.e.
```
Comment (kmill) [inline Mathlib/Analysis/BoxIntegral/Partition/Filter.lean:37]: I wasn't careful with checking `True`/`true` and `False`/`false`, sorry. These ones should be lower case.

Could you check the PRs that I merged and revert any of changes that are terms of `Bool`?

Comment (grunweg) [reply Mathlib/Analysis/BoxIntegral/Partition/Filter.lean:37]: You're right... I just learned about this today. I've checked, only #11948 and this PR exhibit this issue. Hopefully, I can fix these tomorrow.

Comment (grunweg) [reply Mathlib/Analysis/BoxIntegral/Partition/Filter.lean:37]: Actually, this was easier than expected: I filed #11994 for the corrections.

**Pull request closed without merging**